lib/PuppeteerSharp/Launcher.cs

```diff
@@ -40,21 +43,28 @@ public class Launcher
 
         #region Private members
         private Process _chromeProcess;
-        private string _temporaryUserDataDir = null;
-        private Connection _connection = null;
-        private Timer _timer = null;
+        private string _temporaryUserDataDir;
+        private Connection _connection;
+        private Timer _timer;
         private LaunchOptions _options;
         private TaskCompletionSource<bool> _waitForChromeToClose;
-        private static int _processCount = 0;
+        private static int _processCount;
         private bool _processLoaded;
         private const string UserDataDirArgument = "--user-data-dir";
-        private bool _chromiumLaunched = false;
+        private bool _chromiumLaunched;
         #endregion
 
         #region Properties
+        /// <summary>
+        /// Gets or sets a value indicating whether the process created by the instance is closed.
+        /// </summary>
+        /// <value><c>true</c> if is the process is closed; otherwise, <c>false</c>.</value>
         public bool IsChromeClosed { get; internal set; }
         #endregion
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:PuppeteerSharp.Launcher"/> class.
```

Comment:
T: 😉

Reply:
hahaha, it's the IDE! It might know something we don't 😛